### DISCUSSION TITLE
When will the next release occur?

### DISCUSSION THREAD

**Discussion by mattysweeps** [golangci/golangci-lint#2159] [Q&A] [answered]
It's been over a month since the last release, v0.41.1. Since then there's been a lot of linter bumps and other fixes. I'm wondering when a new version will be released?

I checked the documentation but I couldn't find anything about this project's release cadence

**Comment by ldez** [comment, marked as answer]
Hello,

the new version is available https://github.com/golangci/golangci-lint/releases/tag/v1.42.0
